snmp/datadog_checks/snmp/config.py

```diff
@@ -151,10 +151,13 @@ def __init__(
         if not self.metrics and not profiles_by_oid and not profile:
             raise ConfigurationError('Instance should specify at least one metric or profiles should be defined')
 
-        self.scalar_oids, self.next_oids, self.bulk_oids, self.parsed_metrics = self.parse_metrics(self.metrics)
+        parsed_scalar_oids, parsed_next_oids, parsed_bulk_oids, self.parsed_metrics = self.parse_metrics(self.metrics)
```

Comment:
Not sure about the `parsed_` prefix, it seems to only increase verbosity (we know they come from parsing metrics, after all), WDYT?

Reply:
It's a way to distinguish them from the `OIDs` coming from the responses. But yes, the context might be enough.